src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  box-sizing: border-box;
+
+  width: 200px;
```

Comment:
I think it will be more correct to use a variable for the width of the card.

Reply:
agree